database/buildSchema/37_review_list.sql

```diff
@@ -0,0 +1,37 @@
+-- Aggregated VIEW method of all data review data, use in application_list for application list page
```

Comment:
```suggestion
-- Aggregated VIEW method of all related review data to each application on application list page
```

Reply:
thanks, did this as separate commit as my commit before that conflicted 0af288a113c0c0af4aa6c1a329601b2cc43d9a1f